pkg/dev-env/docker-compose.yml

```diff
@@ -66,21 +152,19 @@ services:
       - REDIS_URI=redis://dev-redis:6379/0
```

Comment:
where is the redis running?

Reply:
It's in the same docker-compose file at the top.